### PR TITLE
update type of hits in search response

### PR DESCRIPTION
**Pull request by dichotommy**
This PR is related to https://github.com/meilisearch/documentation/issues/748.

### CODE REVIEW

File: reference/api/search.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Query terms enclosed in double quotes are treated as [phrase searches](#query-q)
 
 | field                | Description                    |    type    |
 | -------------------- | ------------------------------ | :--------: |
-| **hits**             | Results of the query           | `[result]` |
+| **hits**             | Results of the query           | `array of objects` |
```
Comment (maryamsulemani97) [inline reference/api/search.md:62]: Shouldn't you update this for line 150 as well?

Comment (dichotommy) [reply reference/api/search.md:62]: Absolutely right, thanks for catching that!

**Review by maryamsulemani97 — APPROVED** [review]
⭐️

**Comment by maryamsulemani97** [comment]
bors merge

**Pull request merged**